yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -68,6 +68,7 @@ var sourceConnectionFlags = []string{
 type UnsupportedFeature struct {
 	FeatureName string   `json:"FeatureName"`
 	ObjectNames []string `json:"ObjectNames"`
+	DDLs        []string `json:"DDLs"`
```

Comment:
Can we just use ObjectNames when the feature is "UnsupportedDDLs"? I know a DDL is technically not an "object", but for the purpose of this struct, we just want to capture "what it is that is not supported" (featureName), what are the "concerned items that are not supported" (ObjectNames).

Reply:
Hmm, I don't strongly on either of them, I just think this might be readable if we keep ObjectNames and DDLs separately. Let me know which one to choose, I can modify if required.

Reply:
The way I'm thinking about it is: 
For every unsupported feature, you will have EITHER objects OR DDLs, never both. Can you think of any case in which you might have to report both, objects, and their DDLs together? If so, makes sense to keep it. If not, let's re-use object names

Reply:
Let's discuss this tomorrow. Maybe I'm overthinking this, but the DDL field gave me a idea 😅  . 
How about we also include the DDLs for every unsupported feature?
So, for example, you would have "Multi column gin -index" as the section, and you will display both: 
- the name of the index, 
- the CREATE INDEX DDL.

Reply:
Okay sure, will discuss. But I think it will be a lot if we do that like object name and its respective DDL as for a feature if there are lot of objects we will end up repeating similar DDLs for that

Reply:
As discussed, we will include the DDL later in HTML after figuring out the UX for it until then made the changes to include the sqlStmt and object name in a single struct and have that integrated into the Unsupported feature